// @whiskeysockets/baileys/lib/Types/Newsletter.js
export var XWAPaths;
(function (XWAPaths) {
    XWAPaths["xwa2_newsletter_create"] = "xwa2_newsletter_create";
    XWAPaths["xwa2_newsletter_subscribers"] = "xwa2_newsletter_subscribers";
    XWAPaths["xwa2_newsletter_subscribed"] = "xwa2_newsletter_subscribed";
    XWAPaths["xwa2_newsletter_view"] = "xwa2_newsletter_view";
    XWAPaths["xwa2_newsletter_metadata"] = "xwa2_newsletter";
    XWAPaths["xwa2_newsletter_admin_count"] = "xwa2_newsletter_admin";
    XWAPaths["xwa2_newsletter_mute_v2"] = "xwa2_newsletter_mute_v2";
    XWAPaths["xwa2_newsletter_unmute_v2"] = "xwa2_newsletter_unmute_v2";
    XWAPaths["xwa2_newsletter_follow"] = "xwa2_newsletter_follow";
    XWAPaths["xwa2_newsletter_unfollow"] = "xwa2_newsletter_unfollow";
    XWAPaths["xwa2_newsletter_change_owner"] = "xwa2_newsletter_change_owner";
    XWAPaths["xwa2_newsletter_demote"] = "xwa2_newsletter_demote";
    XWAPaths["xwa2_newsletter_delete_v2"] = "xwa2_newsletter_delete_v2";
    XWAPaths["PROMOTE"] = "xwa2_notify_newsletter_admin_promote";
    XWAPaths["DEMOTE"] = "xwa2_notify_newsletter_admin_demote";
    XWAPaths["ADMIN_COUNT"] = "xwa2_newsletter_admin";
    XWAPaths["CREATE"] = "xwa2_newsletter_create";
    XWAPaths["NEWSLETTER"] = "xwa2_newsletter";
    XWAPaths["SUBSCRIBED"] = "xwa2_newsletter_subscribed";
    XWAPaths["METADATA_UPDATE"] = "xwa2_notify_newsletter_on_metadata_update";
})(XWAPaths || (XWAPaths = {}));
export var QueryIdd;
(function (QueryIdd) {
    QueryIdd["METADATA"] = "6620195908089573";
    QueryIdd["GETSUBSCRIBED"] = "6388546374527196";
    QueryIdd["CREATE"] = "6996806640408138";
    QueryIdd["UNMUTE"] = "7337137176362961";
    QueryIdd["MUTE"] = "25151904754424642";
    QueryIdd["FOLLOW"] = "7871414976211147";
    QueryIdd["UNFOLLOW"] = "7238632346214362";
    QueryIdd["UPDATE"] = "7150902998257522";
    QueryIdd["JOB_MUTATION"] = "7150902998257522";
    QueryIdd["ADMIN_COUNT"] = "7130823597031706";
    QueryIdd["CHANGE_OWNER"] = "7341777602580933";
    QueryIdd["DELETE"] = "8316537688363079";
    QueryIdd["DEMOTE"] = "6551828931592903";
    QueryIdd["SUBSCRIBED"] = "6388546374527196";
})(QueryIdd || (QueryIdd = {}));
export var QueryIds;
(function (QueryIds) {
    QueryIds["CREATE"] = "8823471724422422";
    QueryIds["UPDATE_METADATA"] = "24250201037901610";
    QueryIds["METADATA"] = "6563316087068696";
    QueryIds["SUBSCRIBERS"] = "9783111038412085";
    QueryIds["FOLLOW"] = "7871414976211147";
    QueryIds["UNFOLLOW"] = "7238632346214362";
    QueryIds["MUTE"] = "29766401636284406";
    QueryIds["UNMUTE"] = "9864994326891137";
    QueryIds["ADMIN_COUNT"] = "7130823597031706";
    QueryIds["CHANGE_OWNER"] = "7341777602580933";
    QueryIds["DEMOTE"] = "6551828931592903";
    QueryIds["DELETE"] = "30062808666639665";
})(QueryIds || (QueryIds = {}));
export const MexOperations = {
    PROMOTE: 'NotificationNewsletterAdminPromote',
    DEMOTE: 'NotificationNewsletterAdminDemote',
    UPDATE: 'NotificationNewsletterUpdate'
};
